src/components/App.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/no-access-state-in-setstate */
```

Comment:
- [ ] [MANDATORY] Kindly avoid the inline ESLint disabling rule, this is equally the same as altering or changing our linter files. Let us fix it